steward/src/somm_send.rs

```diff
@@ -60,7 +60,9 @@ async fn __send_messages(
     fee: Coin,
     messages: Vec<Msg>,
 ) -> Result<TxResponse, CosmosGrpcError> {
-    let cosmos_address = cosmos_key.to_address(&contact.get_prefix()).unwrap();
+    let cosmos_address = cosmos_key
+        .to_address(&contact.get_prefix())
+        .expect("failed to get address from cosmos key");
```

Comment:
Similarly here, we should probably bubble errors up. It would be preferable to validate configs at startup time rather than later on. For example, if we know we have some Cosmos key we'll be using all the time, we can generate the address for it at config processing time and store it so we aren't constantly re-running these operations and checks.